release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -508,6 +508,11 @@ For more information, see xref:../scalability_and_performance/cnf-numa-aware-sch
 
 For more information about the feature, see xref:../scalability_and_performance/ztp_far_edge/ztp-advanced-policy-config.adoc#ztp-using-pgt-to-configure-power-saving-states_ztp-advanced-policy-config[Configuring power states using PolicyGenTemplates CRs].
 
+[id="ocp-4-13-etcd-overview"]
+==== Overview of etcd is now available
```

Comment:
I'm not sure I've ever seen an announcement of new documentation listed under `New features and enhancements` in release notes. CCing @kalexand-rh to make sure this is where the content would go. 

I also think a more descriptive title could be helpful. Something that highlighted the documentation aspect to help guide the user.

Reply:
The sparse information about etcd has been an issue for long enough that I think putting it in the release notes makes sense - it might be a doc-only enhancement, but it's going to be of at least as much interest as some of the docs that correspond to new product features. (If it did correspond to an engineering feature, we'd drop it here, so I think the placement is ok.)

I think the title is fine, but adding a sentence reminding users how OCP uses etcd might help.

Reply:
I can add a sentence that explains how OCP uses etcd. Thanks @kalexand-rh and @JoeAldinger!